### PR TITLE
Feature/cr 3575 gcp context compute global forwarding rule

### PR DESCRIPTION
**Pull request by lirshindalman**

### CODE REVIEW

File: cloudrail/knowledge/context/gcp/resources/compute/gcp_compute_global_forwarding_rule.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        self.name: str = name
+        self.target: str = target
+
+    def get_keys(self) -> List[str]:
+        return [self.name, self.project_id]
+
+    @property
+    def is_tagable(self) -> bool:
+        return False
+
```
Comment (Imanuel-Mizrahi) [inline cloudrail/knowledge/context/gcp/resources/compute/gcp_compute_global_forwarding_rule.py:27]: @lirshindalman please add: "is_labeled", as far as I see in [TF docs](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_global_forwarding_rule#labels), this resource could have labels (using beta provider for now.)

**Review by Imanuel-Mizrahi — APPROVED** [review]

**Comment by Imanuel-Mizrahi** [comment]
@Tomerva , should we test the "labels" / "tags" fields in the context test for every resource which could have labels/tags?

File: cloudrail/knowledge/context/gcp/gcp_environment_context.py [hunk 2]
```diff
@@ -17,11 +19,13 @@ def __init__(self,
...
-                 projects: AliasesDict[Project] = None):
+                 projects: AliasesDict[Project] = None,
+                 compute_global_forwarding_rule: List[GcpComputeGlobalForwardingRule] = None):
         BaseEnvironmentContext.__init__(self)
         self.checkov_results: Dict[str, List[CheckovResult]] = checkov_results or {}
         self.sql_database_instances: List[GcpSqlDatabaseInstance] = sql_database_instances or []
         self.compute_instances: List[GcpComputeInstance] = compute_instances or []
         self.compute_firewalls: List[GcpComputeFirewall] = compute_firewalls or []
         self.compute_networks: List[GcpComputeNetwork] = compute_networks or []
+        self.compute_global_forwarding_rule: List[GcpComputeGlobalForwardingRule] = compute_global_forwarding_rule or []
```
Comment (Tomerva) [inline cloudrail/knowledge/context/gcp/gcp_environment_context.py:30]: a resource with a unique id should be in an AliasesDict

File: cloudrail/knowledge/context/gcp/resources/compute/gcp_compute_global_forwarding_rule.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    @property
+    def is_labeled(self) -> bool:
+        return True
+
+    def get_id(self) -> str:
```
Comment (Tomerva) [inline cloudrail/knowledge/context/gcp/resources/compute/gcp_compute_global_forwarding_rule.py:32]: is the name unique across projects?

File: tests/knowledge/context/gcp/test_compute_global_forwarding_rule.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+from tests.knowledge.context.gcp_context_test import GcpContextTest
+from tests.knowledge.context.test_context_annotation import context, TestOptions
+
+
+class TestComputeGlobalForwardingRule(GcpContextTest):
+    def get_component(self):
+        return 'compute_global_forwarding_rule'
+
+    @context(module_path="basic", test_options=TestOptions(run_drift_detection=False, run_terraform=False))
```
Comment (Tomerva) [inline tests/knowledge/context/gcp/test_compute_global_forwarding_rule.py:11]: why no terraform and drift tests?

File: tests/knowledge/context/gcp/test_compute_global_forwarding_rule.py [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+    def test_basic(self, ctx: GcpEnvironmentContext):
+        compute = next((compute for compute in ctx.compute_global_forwarding_rule if compute.name == 'test-global-https-forwarding-rule'), None)
+        self.assertIsNotNone(compute)
+        self.assertTrue(compute.target in ['google_compute_global_forwarding_rule.https-forwarding.target', 'https://www.googleapis.com/compute/v1/projects/dev-for-tests/global/targetHttpsProxies/test-proxy'])
```
Comment (Tomerva) [inline tests/knowledge/context/gcp/test_compute_global_forwarding_rule.py:15]: i dont understand this check

**Review by Tomerva — CHANGES_REQUESTED** [review]

**Review by Tomerva — APPROVED** [review]

**Review by Tomerva — APPROVED** [review]

**Pull request merged**